src/elm/clinical.ts

```diff
@@ -99,6 +100,24 @@ export class InValueSet extends Expression {
   }
 }
 
+export class ExpandValueSet extends Expression {
+  valueset: ValueSetRef;
+
+  constructor(json: any) {
+    super(json);
+    this.valueset = new ValueSetRef(json.operand);
+  }
+
+  async exec(ctx: Context) {
+    const valueset = await this.valueset.execute(ctx);
+    if (valueset == null || !valueset.isValueSet) {
+      throw new Error('ValueSet must be provided to ExpandValueSet function');
```

Comment:
This should return a `null` result if the operand result is `null`. If it is not a ValueSet, then it would still make sense for the error to be thrown.

This is defined in the CQL specification. https://cql.hl7.org/04-logicalspecification.html#expandvalueset and https://cql.hl7.org/09-b-cqlreference.html#expandvalueset